starport/services/chain/chain.go

```diff
@@ -111,13 +112,20 @@ func CLIHomePath(path string) Option {
 	}
 }
 
-// KeyringBackend specify the keyring backend to use for the chain command
+// KeyringBackend specifies the keyring backend to use for the chain command
 func KeyringBackend(keyringBackend chaincmd.KeyringBackend) Option {
 	return func(c *Chain) {
 		c.options.keyringBackend = keyringBackend
 	}
 }
 
+// ConfigFile specifies a custom config file to use
+func ConfigFile(configFile string) Option {
+	return func(c *Chain) {
+		c.options.ConfigFile = configFile
```

Comment:
Regarding to https://github.com/tendermint/starport/pull/761#pullrequestreview-591864052,  `c.options.ConfigFile` can be `conf.FileNames` by default and call to `ConfigFile` with a non-empty string will change it to `[]string{configFile}`.

Reply:
I think it's a bit ambiguous. The option should be a single string, `conf.FileNames` is an array because you want to have several possibilities of the file name for the default config